<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formhelp</class>
 <widget class="QWidget" name="Formhelp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>帮助说明</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Gobang.qrc">
    <normaloff>:/Gobang/Image/Icon/图标.jpg</normaloff>:/Gobang/Image/Icon/图标.jpg</iconset>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Gobang/Image/Background/Gme1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="return_Button">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>820</y>
     <width>106</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Gobang/Image/Button/100.40.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="developer_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>690</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>开发人员：何柏坤，胡杨，郝慧峰，朱子潇</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="version_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>750</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>版本：五子棋(星空主题版)V1.3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="help_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>630</width>
     <height>630</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/Gobang/Image/help/1.JPG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="step_1_radioButton">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>660</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="step_2_radioButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>660</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="step_3_radioButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>660</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hint_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 166, 160);</string>
   </property>
   <property name="text">
    <string>点击任一未有棋子的落子点可下棋</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>660</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&gt; &gt; &gt; 点击切换 &gt; &gt; &gt;  </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>660</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  &lt; &lt; &lt; 点击切换 &lt; &lt; &lt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Gobang.qrc"/>
 </resources>
 <connections/>
</ui>
